include/FreeRTOS.h

```diff
@@ -768,7 +768,7 @@
 #endif
 
 #ifndef traceTASK_INCREMENT_TICK
-    #define traceTASK_INCREMENT_TICK( xTickCount )
+    #define traceTASK_INCREMENT_TICK( xTickToIncrement )

```

Comment:
This looks like a typo?

Reply:
xTickCount is actually a global variable in tasks.c, MISRA doesn't allow this kind of condition.

Reply:
Reverted due to suppress all advisory checkers.